app/controllers/OwnerApi.scala

```diff
@@ -0,0 +1,63 @@
+package controllers
```

Comment:
I put all the `Owner` endpoints in another controller/file because it is slightly different from the other resources (no agents, crawling, etc...)